source/run/include/TG4RunManager.h

```diff
@@ -55,12 +55,16 @@ class TG4RunManager : public TG4Verbose
   void LateInitialize();
   void CacheMCStack();
   void ProcessEvent();
+  void ProcessEvent(G4int eventId, G4bool isInterruptible);
   Bool_t ProcessRun(G4int nofEvents);
   Bool_t FinishRun();
 
   // get methods
   Int_t CurrentEvent() const;
   Bool_t SecondariesAreOrdered() const;
+  Bool_t UseExternalGeometryConstruction() const;
+  Bool_t UseExternalParticleGeneration() const;
+  Bool_t IsInterruptibleEvent() const;
```

Comment:
fIsInterruptibleEvent/IsInterruptibleEvent() is defined in TG4EventAction; not needed here